### DISCUSSION TITLE
QGrid and server pagination

### DISCUSSION THREAD

**Discussion by KGBranko** [pratik227/quasar-qgrid#86] [Q&A] [answered]
Hello,

Does QGrid works with server pagination?

**Comment by pratik227** [comment]
> Hello,
> 
> 
> 
> Does QGrid works with server pagination?

Yes

**Reply by KGBranko** [reply]
Thank you for answer.

Nice, that's super to hear.

Any chance you can pinpoint to some code or example?

Thank you in advance.

**Comment by pratik227** [comment, marked as answer]
Github - https://github.com/rafaelameilda/new-q-grid-pagination/blob/master/src/pages/Index.vue

Demo - https://dragged-column-pagination.netlify.app/#/

**Comment by KGBranko** [comment]
Thank you very much for example.

I have managed to solve my problem. I didn't know that q-grid emits "OnRequest" instead of "request" event.  (I did know by analyzing the q-grid code, but forgot about it...)

So, if any1 is using quasar and feathersjs take note to use "OnRequest", not "request" in your q-grid template.

I think this "renamed" event should be listed in your QGrid API, in tab Events, on https://next-quasar-qgrid.netlify.app/

Thank you once again.

**Reply by pratik227** [reply]
Ok Great
